extensions/librdkafka/PublishKafka.cpp

```diff
@@ -43,14 +44,18 @@ core::Property PublishKafka::Topic(core::PropertyBuilder::createProperty("Topic
 
 core::Property PublishKafka::DeliveryGuarantee(
     core::PropertyBuilder::createProperty("Delivery Guarantee")->withDescription("Specifies the requirement for guaranteeing that a message is sent to Kafka")->isRequired(false)
-        ->supportsExpressionLanguage(true)->withDefaultValue("DELIVERY_ONE_NODE")->build());
+        ->supportsExpressionLanguage(true)->withDefaultValue(DELIVERY_ONE_NODE)->build());
 
 core::Property PublishKafka::MaxMessageSize(core::PropertyBuilder::createProperty("Max Request Size")->withDescription("Maximum Kafka protocol request message size")->isRequired(false)->build());
 
 core::Property PublishKafka::RequestTimeOut(
-    core::PropertyBuilder::createProperty("Request Timeout")->withDescription("The ack timeout of the producer request in milliseconds")->isRequired(false)->withDefaultValue<core::TimePeriodValue>(
+    core::PropertyBuilder::createProperty("Request Timeout")->withDescription("The ack timeout of the producer request")->isRequired(false)->withDefaultValue<core::TimePeriodValue>(
         "10 sec")->supportsExpressionLanguage(true)->build());
 
+core::Property PublishKafka::MessageTimeOut(
+    core::PropertyBuilder::createProperty("Message Timeout")->withDescription("The total time sending a message could take")->isRequired(false)->withDefaultValue<core::TimePeriodValue>(
+        "30 sec")->supportsExpressionLanguage(true)->build());
```

Comment:
Please brake this line at "->" operators

Reply:
I agree, but I would refer the honourable member to the previous line and ask him to direct his complaints to the original author. (Will fix.)